File: SQL/ISQLStatement.vb

' ___________________________________________________
'
'  © Hi-Integrity Systems 2010. All rights reserved.
'  www.hisystems.com.au - Toby Wicks
' ___________________________________________________
'

Option Strict On
Option Explicit On 

Namespace SQL

    Public Interface ISQLStatement

        Property ConnectionType() As Database.ConnectionType
        ReadOnly Property SQL() As String

    End Interface

End Namespace

File: SQL/ISQLStatement.cs
// ___________________________________________________
//
//  © Hi-Integrity Systems 2010. All rights reserved.
//  www.hisystems.com.au - Toby Wicks
// ___________________________________________________
//

using System.Collections;
using System;
using System.Data;

namespace DatabaseObjects.SQL
{
	public interface ISQLStatement
	{
		Database.ConnectionType ConnectionType { get; set; }
		string SQL { get; }
	}
}
